### ISSUE TITLE
Cannot get to my files

### ISSUE THREAD

**Issue by KowalskiBio** [zhxiaokang/RASflow#40]
Hello, 
I am starting to use your program and I cannot get  it to work. The directory is correct, when I go there via cd command and check content via ls, it writes my fastq files (SRRnumber_1/2.fastq.gz. So I know they are there. What am I missing? 
Thanks for reply

cp: cannot stat '/auto/kowalski/fastq/SRR8750770_*R1*.f*q.gz': No such file or directory
[Fri Jul 26 19:04:19 2024]
Error in rule getReads:
    jobid: 0
    output: data/output/test2/reads/SRR8750770_forward.fastq.gz, data/output/test2/reads/SRR8750770_reverse.fastq.gz

RuleException:
CalledProcessError in line 24 of /auto/brno2/home/kowalski/RASflow/workflow/quality_control.rules:
Command ' set -euo pipefail;  scp -i None auto/kowalski/fastq/SRR8750770_*R1*.f*q.gz data/output/test2/reads/SRR8750770_forward.fastq.gz ' returned non-zero exit status 1.
  File "/auto/kowalski/RASflow/workflow/quality_control.rules", line 24, in __rule_getReads
  File "/storage/kowalski/miniconda3/envs/rasflow/lib/python3.6/concurrent/futures/thread.py", line 56, in run
Exiting because a job execution failed. Look above for error message

My metadata file:
sample	group	subject
SRR27494859	accumbens	Hancock
SRR27494858	accumbens	Hancock
SRR27494754	dlpc	Hancock
SRR27494755	dlpc	Hancock
SRR8750749	caudate	Khrameeva
SRR8750750	caudate	Khrameeva
SRR8750751	caudate	Khrameeva
SRR8750752	caudate	Khrameeva
SRR8750753	accumbens	Khrameeva
SRR8750754	accumbens	Khrameeva
SRR8750755	accumbens	Khrameeva
SRR8750756	accumbens	Khrameeva
SRR8750769	putamen	Khrameeva
SRR8750770	putamen	Khrameeva
SRR8750771	putamen	Khrameeva
SRR8750772	putamen	Khrameeva
SRR8750785	globusp	Khrameeva
SRR8750786	globusp	Khrameeva
SRR8750787	globusp Khrameeva
SRR8750788	globusp	Khrameeva
SRR8750797	ec	Khrameeva
SRR8750798	ec	Khrameeva
SRR8750799	ec	Khrameeva
SRR8750800	ec	Khrameeva
SRR25755687	striatum	Swchwamborn
SRR25755688	midbrain	Swchwamborn

**Comment by KowalskiBio** [comment]
I should add that I tried to run the example and It worked just fine.

**Comment by zhxiaokang** [comment]
Could you `ls` your directory so that I can see the exact file names. As you wrote 'SRRnumber_1/2.fastq.gz' and there isn't "R" before '1/2', but the rule actually looks for 'R' as states "/auto/kowalski/fastq/SRR8750770_R1.f*q.gz"

**Comment by KowalskiBio** [comment]
I am very sorry, the cause of error was rather hidden and on my side. My security keys were not working properly, therefore I couldn't use scp -i. When I changed it to cp and deleted the key portion, it started working. I have another question, but my former question was resolved.
I should add that I had to edit the script and remove the R in "_R1" and "_R2", because all of my files are without the R, but that was fast and simple. Thanks for your help!
